### PR TITLE
fix build instructions

### PR DESCRIPTION
**Pull request by DioEgizio**

### CODE REVIEW

File: src/wiki/development/build-instructions.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Getting the project to build and run on Linux is easy if you use any modern and
 ### Build dependencies
 - A C++ compiler capable of building C++11 code.
 - Qt Development tools 5.12 or newer (`qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools libqt5core5a libqt5network5 libqt5gui5` on Debian-based systems)
-  - Alternatively you can also use Qt 6.0 or newer (`qt6-base-dev qtchooser qt6-base-dev-tools libqt6core6 libqt6core5compat6 libqt6network6 libqt6network6` on Debian (testing/unstable) based systems), if you prefer it.
+  - Alternatively you can also use Qt 6.0 or newer (`qt6-base-dev qtchooser qt6-base-dev-tools libqt6core6 libqt6core5compat6 libqt6network6 libqt6network6` on Debian (testing/unstable), Ubuntu 22.04 based systems (note: on Ubuntu, Qt 6 is available only on 22.04 or higher in official repositories, however you can use [ppa:okirby/qt6-backports])https://bugs.launchpad.net/~okirby/+archive/ubuntu/qt6-backports) on older versions)), if you prefer it.
```
Comment (Scrumplex) [inline src/wiki/development/build-instructions.md:36]: Too much information IMO.

Comment (DioEgizio) [reply src/wiki/development/build-instructions.md:36]: Ye probably, maybe that part should be removed

File: src/wiki/development/build-instructions.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Getting the project to build and run on Linux is easy if you use any modern and
 ### Build dependencies
 - A C++ compiler capable of building C++11 code.
 - Qt Development tools 5.12 or newer (`qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools libqt5core5a libqt5network5 libqt5gui5` on Debian-based systems)
-  - Alternatively you can also use Qt 6.0 or newer (`qt6-base-dev qtchooser qt6-base-dev-tools libqt6core6 libqt6core5compat6 libqt6network6 libqt6network6` on Debian (testing/unstable) based systems), if you prefer it.
+  - Alternatively you can also use Qt 6.0 or newer (`qt6-base-dev qtchooser qt6-base-dev-tools libqt6core6 libqt6core5compat6 libqt6network6 libqt6network6` on Debian), if you prefer it.
```
Comment (Scrumplex) [inline src/wiki/development/build-instructions.md:36]:
```suggestion
  - Alternatively you can also use Qt 6.0 or newer (`qt6-base-dev qtchooser qt6-base-dev-tools libqt6core6 libqt6core5compat6 libqt6network6 libqt6network6` on Debian (testing/unstable) based systems), if you prefer it.
```

**Review by Scrumplex — APPROVED** [review]

**Comment by Scrumplex** [comment]
What the fuck. I merged this???

**Comment by Scrumplex** [comment]
https://github.com/PolyMC/polymc.github.io/commit/92d445ccc099d913ec0bf5578380d7c7abf383df

Why is the PR still open???

**Comment by Scrumplex** [comment]
GitHub moment

**Pull request closed without merging**